src/Salt/UserBundle/Controller/SignupController.php

```diff
@@ -65,6 +65,16 @@ public function signupAction(Request $request)
                 $command = new AddUserCommand($targetUser, $encryptedPassword);
                 $this->sendCommand($command);
 
+                // email the new user
+                $email = $targetUser->getUsername();
+                $message = (new \Swift_Message('Hello Email'))
```

Comment:
Should this be checking the `USE_MAIL_FEATURE` option being passed?

Reply:
This is now checking for the email feature being active